### DISCUSSION TITLE
Piston Animation

### DISCUSSION THREAD

**Discussion by HollowedWanderer** [ThePandaOliver/Pandas-Extra-Details#1] [Ideas]
Hey! I saw your mod and it looks really cool! I've been looking for a mod that changes piston animations just to add a little more life into them like you did with the doors and trapdoors. I know you're open to new so ideas so I was thinking it might be a cool idea to change the pistons a little, that's all. Thanks for the cool mod!

**Comment by ThePandaOliver** [comment]
i will see what i can do

**Comment by Andrew6rant** [comment]
[Telepistons](https://modrinth.com/mod/telepistons) can do this (works very well when paired with [Vectorientation](https://modrinth.com/mod/vectorientation)).

Both are in the public domain so you could adapt the code to be multi-loader if you wish to
